locales/en.json

```diff
@@ -35,7 +35,7 @@
     "errorUpdate": "Something went wrong, are you connected to the internet? {errorMessage}"
   },
   "AvatarUploader": {
-    "errorAvatarUpload": "Could not upload image. Is it the right format and size?"
```

Comment:
Think better with 1 MB rather than 1MB? (with space) :)

Reply:
I am not sure if "Max" is correct English grammar but maybe "...png with a maximum size of 1 MB"